packages/flutter/test/widgets/container_test.dart

```diff
@@ -323,6 +323,7 @@ void main() {
       box.toStringDeep(minLevel: DiagnosticLevel.hidden),
```

Comment:
Feel free to instead remove the DiagnosticLevel.hidden from this toStringDeep call. That just makes the output of this test more confusing.

Reply:
OK, That's what I'll do.

Reply:
Actually, it looks like that test is there specifically testing the different levels of diagnostic output for `Container`.  There are tests for each of the levels, so if I change this, I would just get rid of it (since there's already a test for each of the others).  It doesn't make sense to get rid of it, so I'm just going to leave it as-is.